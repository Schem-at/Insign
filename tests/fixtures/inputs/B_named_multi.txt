@dataloop=rc([0,0,0],[31,7,15])
#dataloop:type="accumulator"

@dataloop=rc([32,0,0],[63,7,15])
#performance="fast"

@cache=ac([100,100,100],[103,103,103])
#cache:level=2
