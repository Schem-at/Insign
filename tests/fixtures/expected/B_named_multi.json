{
  "cache": {
    "bounding_boxes": [
      [
        [
          100,
          100,
          100
        ],
        [
          103,
          103,
          103
        ]
      ]
    ],
    "metadata": {
      "level": 2,
      "performance": "fast"
    }
  },
  "dataloop": {
    "bounding_boxes": [
      [
        [
          0,
          0,
          0
        ],
        [
          31,
          7,
          15
        ]
      ],
      [
        [
          32,
          0,
          0
        ],
        [
          63,
          7,
          15
        ]
      ]
    ],
    "metadata": {
      "type": "accumulator"
    }
  }
}